shared/clinical-code-sets/medications/statins/1/statins.snomed.txt

```diff
@@ -0,0 +1,846 @@
+42297611000001103	Simvastatin 80mg / Ezetimibe 10mg tablets
+4896711000001108	Simvador 40mg tablets (Dexcel-Pharma Ltd)
+13886111000001107	Simvastatin 40mg/5ml oral suspension (Special Order) (product)
+20970211000001101	Atorvastatin 60mg tablets (Alliance Healthcare (Distribution) Ltd) (product)
+15158611000001106	Simvador 80mg tablets (Dexcel-Pharma Ltd)
+42382111000001102	Simvastatin 20mg tablets
+22047511000001102	Atorvastatin 20mg tablets (Almus Pharmaceuticals Ltd) (product)
+41139011000001108	Atorvastatin 10mg tablets (Amarox Ltd)
+29772511000001108	Atorvastatin 40mg tablets (Sigma Pharmaceuticals Plc) (product)
+18467811000001106	Simvastatin 10mg tablets (Accord-UK Ltd)
+14951711000001101	Pravastatin 5mg/5ml oral suspension (Special Order) (product)
+13831811000001101	Atorvastatin 40mg tablets (DE Pharmaceuticals)
+17841911000001100	Simvastatin 20mg/5ml oral suspension sugar free (Alliance Healthcare (Distribution) Ltd) (product)
+320037003	Cerivastatin 300microgram tablets
+32394811000001103	Atorvastatin 20mg tablets (Kent Pharma (UK) Ltd)
+13923011000001100	Lipitor 20mg tablets (DE Pharmaceuticals)
+10295411000001101	Simvastatin 40mg tablets (Focus Pharmaceuticals Ltd) (product)
+408024009	Rosuvastatin 40mg tablets
+35136211000001107	Atorvastatin 20mg tablets (Milpharm Ltd) (product)
+414177002	Simvastatin 20mg / Ezetimibe 10mg tablets
+16198611000001101	Lescol 40mg capsules (Lexon (UK) Ltd) (product)
+15171911000001107	Simvastatin 20mg tablets (Sigma Pharmaceuticals Plc) (product)
+14079011000001107	Atorvastatin 40mg/5ml oral suspension (Special Order) (product)
+14036711000001102	Fluvastatin 40mg capsules (Zentiva Pharma UK Ltd)
+30056011000001104	Fluvastatin 80mg modified-release tablets (DE Pharmaceuticals) (product)
+19733611000001101	Simvastatin 40mg tablets (Medreich Plc) (product)
+42302911000001100	Atorvastatin 20mg tablets (Medreich Plc)
+36734011000001101	Rosuvastatin 40mg tablets (Macleods Pharma UK Ltd) (product)
+320023007	Fluvastatin 40mg capsules
+42618311000001107	Simvastatin 80mg / Ezetimibe 10mg tablets (Alliance Healthcare (Distribution) Ltd) (product)
+20496611000001108	Atorvastatin 20mg tablets (Aspire Pharma Ltd) (product)
+320036007	Cerivastatin 200microgram tablets
+34954211000001108	Rosuvastatin 20mg tablets (Aristo Pharma Ltd)
+41548011000001105	Atorvastatin 40mg tablets (Brown & Burk UK Ltd)
+9316111000001109	Simvastatin 80mg / Ezetimibe 10mg tablets (product)
+9316011000001108	Simvastatin 40mg / Ezetimibe 10mg tablets (product)
+4172611000001100	Rosuvastatin 40mg tablets (product)
+4172511000001104	Rosuvastatin 20mg tablets (product)
+9315911000001100	Simvastatin 20mg / Ezetimibe 10mg tablets (product)
+4172411000001103	Rosuvastatin 10mg tablets (product)
+32170911000001100	Cholib 145mg/20mg tablets (Viatris UK Healthcare Ltd) (product)
+32234211000001101	Fenofibrate 145mg / Simvastatin 20mg tablets (product)
+42209911000001100	Fenofibrate micronised 160mg tablets (product)
```

Comment:
Fenofibrate is not a static. Cholib (fenofibrate & simvastatin) is fine, but the ones that are just fenofibrate should be removed.